pynicotine/userinfo.py

```diff
@@ -118,7 +118,7 @@ def _user_info_request(self, msg):
         self.requested_info_times[user] = request_time
 
         if core.login_username != user:
-            log.add(_("User %(user)s is reading your user info"), {'user': user})
+            log.add(_("User %(user)s is viewing your user profile"), {'user': user})
```

Comment:
Potentially ambiguous and superfluous use of the word "user" here.

`"User %(user)s is viewing your profile"`